starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,63 @@
+body {
+    font-family: sans-serif;
+}
+
+div {
```

Comment:
Avoid using element selectors (div, p, ul, h1, ol...), better use classes and ids.